openmedia/player/mixer.py

```diff
@@ -0,0 +1,108 @@
+from pygame import mixer
+from track import Track
+import os
+
+class InvalidVolumeError(Exception):
+    pass
+
+track_list = []
+song_index = 0
+total_tracks = 0
+current_song = None
+is_paused = False
+is_stopped = False
+offset = 0
+
+def init(song_list):
+    global track_list, song_index, total_tracks, current_song, is_paused
+    track_list = [Track(song) for song in song_list]
+    mixer.init()
+    set_volume(0.5)
+    song_index = -1
+    total_tracks = len(track_list)
+    current_song = _get_next_song()
+    is_paused = False
+
+def play(filename=None):
```

Comment:
You should rename `filename` to something else because you are actually passing in the full path of a song.